### DISCUSSION TITLE
MaxBarWidth not working (for me)

### DISCUSSION THREAD

**Discussion by Szoldzsi** [Live-Charts/LiveCharts2#1192] [Q&A]
Hi! I have made a chart, where the X axis are dates, and the Y axis are values connected to those dates. Everything shows up perfectly, except that i cant change the bar width, and all columns are tiny even when i zoom in.

This is the code for the creation of the chart:

```
        public Form1()
        {
            InitializeComponent();
            
            cartesianChart1.Series = new ISeries[]
            {
                new ColumnSeries<DateTimePoint>
                {
                    Values = GetChartData(),
                    MaxBarWidth = 10,
                    Padding = 1

                }
            };
            var axisX = new Axis
            {
                Labeler = val => new System.DateTime((long)val).ToString("yyyy. MM. dd"),
                MinStep = 10,
            };
            cartesianChart1.ZoomMode = LiveChartsCore.Measure.ZoomAndPanMode.X;
            cartesianChart1.XAxes = new List<Axis> { axisX };
        }
```

And this is what appears, when i start the program:

![image](https://github.com/beto-rodriguez/LiveCharts2/assets/116651949/2ed79279-0438-4f82-b3d5-ef47d9b5076a)

Thank you in advance!

**Comment by hdlee** [comment]
according to my experience, u need to set UnitWidth as well.
